js/arena.js

```diff
@@ -0,0 +1,95 @@
+import gladiator from './gladiator.js';
+
+
+
+export default class arena extends gladiator {
+
+    constructor(name) {
+
+        this.name = name.charAt(0).toUpperCase() + name.substring(1);
+
+        this.gladiators = [];
+    }
+
+ 
+
+    addGladiator(gladiator) {
+
+        if (this.gladiators.length < 2) {
+            this.gladiators.push(Gladiator)
+        }
+
+    }
+
+
+    removeGladiator(gladiator) {
+
+        if (this.gladiators.length < 2) {
+            this.gladiators.pop(Gladiator)
```

Comment:
.pop() does not work here. pop is a method does not take any arguments

one way to remove an element is to use a method called .splice() this method will help remove specific element from an array 
here is a link on how to use it
[splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)